javascript-practice/src/pages/dashboard.html

```diff
@@ -95,6 +97,99 @@ <h2 class="user-dashboard-heading">Users Dashboard</h2>
           </div>
         </div>
       </section>
+      <div class="popup-overlay">
```

Comment:
You use div too much, consider use html5 tags as much as you can

Reply:
I'll try use more HTML5 tags